portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,17 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds a random fact.
+ */
+function addRandomFact() {
+    const facts = ["I have no sibling", "I seldom eat spicy food", "I don't like watching thrillers"];
+    
+    // Pick a random fact.
+    const fact = facts[Math.floor(Math.random() * facts.length)];
```

Comment:
Just to confirm. Do we need to check if that fact has been picked before or we are just showing one fact at a time (aka replacing existing one instead of appending) and showing old one is ok?

Reply:
the requirement does not specify it, but i think maybe add such check will be better

Reply:
Fixed